.rspec

```diff
@@ -1,2 +1 @@
 --require spec_helper
---format documentation
```

Comment:
I like this format — must we remove it from the defaults? Couldn't we override in the rake task?

Reply:
👍